src/StoreForwarding.cpp

```diff
@@ -1222,17 +1223,17 @@ class RenameVars : public IRMutator {
     }
 };
 
-vector<Store> find_non_aliasing_stores(Stmt stmt) {
+vector<const Store *> find_non_aliasing_stores(Stmt stmt) {
```

Comment:
The usage here is fine because the results of this function are used while stmt stays in scope. However, it's a big dangerous because the result doesn't increment the reference count of the stores.

Is there a reason you changed this to be pointers instead of values?